docker-compose.yml

```diff
@@ -1,13 +1,23 @@
 version: "2.2"
 
 services:
+  nginx:
```

Comment:
As I said before, we're really low on memory on the server, so I'm worried with another Docker image.

- What's the advantage of having a separate Docker nginx image?

- Could we instead run Nginx inside dash_server? Would this save memory?